packages/chaire-lib-backend/src/config/__tests__/ServerConfig.test.ts

```diff
@@ -52,7 +52,29 @@ describe('get routing mode and engine configs', () => {
     });
 
     test('getRoutingEngineConfigForMode, mode does not exist', () => {
-        expect(ServerConfig.getRoutingEngineConfigForMode('walking', 'osrm')).toBeUndefined();
+        expect(ServerConfig.getRoutingEngineConfigForMode('walking_way_data_as_name', 'osrmRouting')).toBeUndefined();
     });
 
+    test('getAllModesForEngine, engine exists', () => {
```

Comment:
Just nit picking on the test name here. 

The test validate that the modes have a specific default engine, not necessary that they just exist. We might want to ajust the name or the test